### PR TITLE
more queries for the Fuzzer

### PR DESCRIPTION
**Pull request by mgarciaberlin**

### CODE REVIEW

File: external-resolver/src/test/scala/org/genivi/sota/resolver/test/Random.scala [hunk 1]
```diff
@@ -38,8 +38,14 @@ class Random extends ResourcePropSpec {
               case Success               => ()
               case SuccessVehicles(vehs) => responseAs[Set[Vehicle]]                       shouldBe vehs
               case SuccessPackage(pkg)   => responseAs[Package]                            shouldBe pkg
-              case SuccessPackages(pkgs) => responseAs[Set[PackageId]]                     shouldBe pkgs
+
+              // TODO Package.description roundtripped wrongly for non-Latin chars. Sidestep by comparing PackageIds.
```
Comment (stevana) [inline external-resolver/src/test/scala/org/genivi/sota/resolver/test/Random.scala]: Is there a ticket for this? (So we don't forget it.)

File: external-resolver/src/test/scala/org/genivi/sota/resolver/test/random/Query.scala [hunk 2]
```diff
@@ -103,14 +133,30 @@ object Query extends
...
+          Store.pickVehicleWithComponent.runA(s) map { case (veh, cmp) => ListVehiclesFor(cmp) }),
 
-        (if (vehs > 0) 10 else 0,
-          Store.pickVehicle.runA(s).map(ListPackagesOnVehicle(_))),
+        (if (pfilt > 0) 10 else 0,
+          Store.pickPackageWithFilter.runA(s) map { case (pkg, flt) => ListPackagesFor(flt) }),
 
-        (5,  Gen.const(ListFilters)),
+        (if (pkgs > 0) 10 else 0,
+          Store.pickPackage.runA(s).map(ListFiltersFor(_))),
```
Comment (stevana) [inline external-resolver/src/test/scala/org/genivi/sota/resolver/test/random/Query.scala]: I suppose this list will be empty very often. Does it make sense to use pickPackageWithFilter here as well?

File: external-resolver/src/test/scala/org/genivi/sota/resolver/test/Random.scala (hunk 1 above)
Comment (mgarciaberlin) [inline external-resolver/src/test/scala/org/genivi/sota/resolver/test/Random.scala]: https://advancedtelematic.atlassian.net/browse/PRO-314

File: external-resolver/src/test/scala/org/genivi/sota/resolver/test/random/Query.scala (hunk 2 above)
Comment (mgarciaberlin) [inline external-resolver/src/test/scala/org/genivi/sota/resolver/test/random/Query.scala]: Good point, addressed in https://github.com/advancedtelematic/rvi_sota_server/pull/340/commits/d8f7356acc2f5a9a025e10a93ab337ca367ba8c8

**Comment by stevana** [comment]
Nice, feel free to squash and merge once check pass.

**Pull request merged**